pkg/api/client/client.go

```diff
@@ -95,7 +94,6 @@ func NewDefault(addr string, opts ...Option) *DefaultClient {
 		client:   http.DefaultClient,
 		scheme:   "http://",
 		hostAddr: addr,
-		timeout:  defaultRequestTimeout,
```

Comment:
This would essentially tank any distributed query running longer than 30 seconds

Reply:
As discussed: Please add back some (very long) default timeout to avoid stale connections due to misuse of the client by people... :wink: